client-misk-jooq/src/main/kotlin/app/cash/backfila/client/misk/jooq/BackfillJooqTransacter.kt

```diff
@@ -0,0 +1,10 @@
+package app.cash.backfila.client.misk.jooq
+
+import org.jooq.DSLContext
+
+interface BackfillJooqTransacter {
+  fun <RETURN_TYPE> transaction(
```

Comment:
I think it's more common to just use a single letter like  `R`

Reply:
I don't like using single letter generic parameters. It's very cryptic and hard to understand.  I wrote a blog about it once - https://medium.com/@readsethu/java-generics-and-why-has-conventions-made-it-confusing-to-use-d9afb16c49ec